bin/node/runtime/Cargo.toml

```diff
@@ -50,6 +54,7 @@ frame-try-runtime = { version = "0.10.0-dev", default-features = false, path = "
 pallet-authority-discovery = { version = "4.0.0-dev", default-features = false, path = "../../../frame/authority-discovery" }
 pallet-authorship = { version = "4.0.0-dev", default-features = false, path = "../../../frame/authorship" }
 pallet-babe = { version = "4.0.0-dev", default-features = false, path = "../../../frame/babe" }
+pallet-bags-list = { version = "4.0.0-dev", default-features = false, path = "../../../frame/bags-list" }
```

Comment:
Why do we need this?

Reply:
Check details [here](https://www.notion.so/cere/Testing-logs-1f7d5107060a432398bad8b6a15225f0#55e0d39b91ef457783aef353f6afba97)